ndcube/asdf/converters/ndcube_converter.py

```diff
@@ -39,9 +39,14 @@ def to_yaml_tree(self, ndcube, tag, ctx):
             This ensures that users are aware of potentially important information
             that is not included in the serialized output.
         """
+        from astropy.wcs.wcsapi import HighLevelWCSWrapper
+
         node = {}
         node["data"] = ndcube.data
-        node["wcs"] = ndcube.wcs
+        if isinstance(ndcube.wcs, HighLevelWCSWrapper):
+            node["wcs"] = ndcube.wcs._low_level_wcs
```

Comment:
```suggestion
        if isinstance(ndcube.wcs, BaseHighLevelWCS):
            node["wcs"] = ndcube.wcs.low_level_wcs
```

We can just do this as all high level objects have to have the `low_level_wcs` property and classes which implement both the high and low level APIs just return self. (Like astropy.wcs and gwcs)